plugins/utils/notify.nix

```diff
@@ -7,11 +7,20 @@
 with lib; let
   cfg = config.plugins.notify;
   helpers = import ../helpers.nix {inherit lib;};
+  optionWarnings = import ../../lib/option-warnings.nix {inherit lib;};
+  basePluginPath = ["plugins" "notify"];
   icon = mkOption {
     type = types.nullOr types.str;
     default = null;
   };
 in {
+  imports = [
+    (optionWarnings.mkRenamedOption {
```

Comment:
Could you add a comment of the date, in case we want to cleanup

Reply:
Comment added